.github/workflows/production-workflow.yml

```diff
@@ -0,0 +1,107 @@
+name: production
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build_images:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        app:
+          - backend
+          - client
+          - geo-service
```

Comment:
yeah i guess by the time we will deploy to production we won't need the python service anymore since @joshmorel is working on that and putting it into node

Reply:
Yeah I know, I was going to ask about removing it but didn't want to break things